src/hotspot/share/runtime/escapeBarrier.hpp

```diff
@@ -80,7 +81,7 @@ class EscapeBarrier : StackObj {
   // Revert ea based optimizations for all java threads
   EscapeBarrier(bool barrier_active, JavaThread* calling_thread)
     : _calling_thread(calling_thread), _deoptee_thread(NULL),
-      _barrier_active(barrier_active && (JVMCI_ONLY(UseJVMCICompiler) NOT_JVMCI(false)
+      _barrier_active(barrier_active && !Continuations::enabled() && (JVMCI_ONLY(UseJVMCICompiler) NOT_JVMCI(false)
                       COMPILER2_PRESENT(|| DoEscapeAnalysis)))
   {
```

Comment:
Will need to double-check but I doubt the changes in escapeBarrier.hpp are really needed.
Otherwise, how can the escapeBarrier's be needed while the EA is disabled?